### DISCUSSION TITLE
Why the releases of ente photos & auth are in the same repo?

### DISCUSSION THREAD

**Discussion by hrshtein** [ente-io/ente#1087] [Q&A] [answered]
I am using obtainium for updates, and i have added the repo only for auth updates but currently obtainium is showing update for photos!

**Comment by mnvr** [comment]
They're in the same repo since we switched to a single monorepo when we [open sourced our server](https://ente.io/blog/open-sourcing-our-server/).

If you want to filter them, you can use the tag prefixes:

* Auth releases will be always of a format "auth-v1.2.3"
* Photos releases will always be of a format "photos-v1.2.3"

**Comment by vishnukvmd** [comment]
Hey, we've consolidated all of our code into a monorepo, so releases will be pushed from here.

There seems to be a way within Obtanium to filter releases by regex. Auth releases will be prefixed with `auth-v2.`: https://github.com/ente-io/ente/releases?q=tag%3Aauth-v2

Could you please try that out?

**Reply by hrshtein** [reply]
added this but still obtainium is suggesting photos app, @ImranR98 please suggest!

**Reply by ImranR98** [reply, marked as answer]
@hrshtein there is no tag filter in Obtainium, but you can filter by release titles instead (for this repo it looks like tags are identical to releases so that's convenient). Read the Wiki for more details on filtering options.

Note you have to add the filter in Obtainium's settings, adding the URL above won't make a difference.

**Reply by hrshtein** [reply]
Thanks, it worked
